setup.py

```diff
@@ -14,6 +15,27 @@
 
 from setuptools import find_packages
 from setuptools import setup
+from setuptools.command.test import test as TestCommand
+
+
+class PyTest(TestCommand):
```

Comment:
Basically, this class doesn't add much usability, it's just an indirection to py.test. If you want to reduce some lines of code you can remove these. Otherwise leave it. ;-)